tests/unit/data/old/config_out.json

```diff
@@ -20,15 +21,17 @@
           "y": "200",
           "width": 200,
           "height": 100,
-          "uid": "Zone 1"
+          "uid": "Zone 1",
+          "align": "center"
```

Comment:
@e-kolpakov Ah, right -- I thought I was commenting on one of the `data.json` files :) Sorry for the noise.